src/front-end/src/components/MentorForm.js

```diff
@@ -105,6 +152,50 @@ class MentorForm extends Component {
                 }
               />
             </Form.Item>
+            <Form.Item
+              label="Top 3 skills"
+              name="skill list"
+              rules={[
+                { required: true, message: 'Please choose your best skill' },
+              ]}
+              tooltip="What skill can you help people learn?"
+            >
+              <select id="best">
+                <option value="" disable selected>
+                  Select your option
+                </option>
+                <option>1</option>
+                <option>2</option>
+                <option>3</option>
+                <option>4</option>
+              </select>
+              &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
+              <select id="second">
+                <option value="" disable selected>
+                  Select your option
+                </option>
+                <option>1</option>
+                <option>2</option>
+                <option>3</option>
+                <option>4</option>
+              </select>
+              &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
+              <select id="third">
+                <option value="" disable selected>
+                  Select your option
+                </option>
+                <option>1</option>
+                <option>2</option>
+                <option>3</option>
+                <option>4</option>
+              </select>
+            </Form.Item>
+            <Form.Item name="description" label="Description">
+              <Input.TextArea
+                id="description"
+                placeholder="Tell us a bit about yourself"
+              />
+            </Form.Item>
```

Comment:
This field should be required, and included in the Mentee as form as well.

Reply:
done